app/models/role.rb

```diff
@@ -1,11 +1,16 @@
 #NOTE add the person object you want to attach role to...
+
 class Role < ActiveRecord::Base
   belongs_to :person
 
+  validates :person, presence: true
+  validates :name, uniqueness: {scope: :person_id}
+  validates :name, inclusion: { in: %w(admin spotlight) }
+
   scope :admins, -> { where(name: 'admin') }
 
   def self.is_admin?(person)
-    find_by_person_id_and_name(person.id, 'admin')
+    self.exists?(person_id: person.id, name: 'admin')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.